assets/css/map/markers/_missed_stop_icon.scss

```diff
@@ -0,0 +1,72 @@
+@use "../../color/definitions" as semantic;
+
+.c-missed-stop-icon {
+  transition: 50ms 50ms;
+
+  &,
+  svg {
+    display: block;
+    overflow: visible;
+  }
+
+  &,
+  svg,
+  .c-missed-stop-icon__circle,
+  .c-missed-stop-icon__line,
+  .c-missed-stop-icon__focus-circle {
+    transform-origin: center;
+  }
+
+  .c-missed-stop-icon__circle {
+    fill: $white;
+    stroke-width: 1px;
+  }
+
+  .c-missed-stop-icon__focus-circle {
+    stroke: transparent;
+  }
+
+  .c-missed-stop-icon__line {
+    stroke-width: 2px;
+  }
+
+  .c-missed-stop-icon__circle,
+  .c-missed-stop-icon__line {
+    stroke: semantic.$missed-stop;
+  }
+}
+
+.c-missed-stop-icon-container:not(
+    .c-missed-stop-icon-container--interactions-disabled
+  ) {
+  &:focus-visible {
+    outline-offset: 2px;
+  }
+
+  &:hover .c-missed-stop-icon,
+  &:focus .c-missed-stop-icon,
+  .c-missed-stop-icon[data-selected] {
+    scale: 1.2;
+
+    .c-missed-stop-icon__circle {
+      stroke-width: 2px;
+    }
+
+    .c-missed-stop-icon__line {
+      fill: $color-eggplant-700;
+    }
+  }
+
+  &:focus .c-missed-stop-icon {
+    .c-missed-stop-icon__focus-circle {
+      stroke: $color-eggplant-500;
```

Comment:
**question:** do we want to be using the old 2021 color palette, or the 2024 palette? I think for the detour stuff in general we've been using the 2024 colors because in some places the the 2021 colors look noticeably off.

Reply:
WUPS! Yeah let's try to use the right color here.

...and maybe for regular stop icons too?

Reply:
Actually, thinking this through for a moment... our options are:
1. Leave it as-is for now (it may look a bit out of place because of the 2021/2024 colors)
2. Change it only for missed stops (that might look out of place because then regular stops and missed stops will look different)
3. Change it for regular stops AND missed stops (now we're changing existing stuff, which is... probably fine because we want to move towards 2024 colors anyway, but still a bit risky)

Doesn't really seem like we have a great answer here. I think my preference is to do option 1, and have a quick design/dev discussion about whether (and when) we should change both missed stops and regular stops to the 2024 color.

Reply:
I like 1, agree that changing existing stuff is not something we should _"just do"_

Reply:
Agreed that 1 is fine, 2 would also be fine but either way agreed that a quick design / dev discussion is warranted.

Reply:
[Asana ticket to follow up on this](https://app.asana.com/0/1148853526253420/1206798268447431/f)